### PR TITLE
feat: Implementation calendrier statique + affiche évènement

### PR DESCRIPTION
**Pull request by tcht-marie**
## Checklist
- [x] I didn't over-scope my PR
- [x] My PR title matches the [commit convention](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I did not include breaking changes
- [x] I made my own code-review before requesting one

### I included unit tests that cover my changes
- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### I added/updated the documentation about my changes
- [ ] 📜 README.md
- [ ] 📕 docs/*.md
- [ ] 📓 other
- [x] 🙅 no documentation needed

### CODE REVIEW

File: java/src/main/java/io/simplon/toomanychoco/App.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public void start(int numberOfThreads) throws IOException {
 
 		// Create a basic handler for index route ('/')
 		server.createContext("/", (request -> {
-			String response = "Hello, world!";
+			String response = "bye bye!";
```
Comment (shiipou) [inline java/src/main/java/io/simplon/toomanychoco/App.java:43]: I think this change overscope your PR
```suggestion
			String response = "Hello, world!";
```

File: react-app/package.json [hunk 2]
```diff
@@ -1,12 +1,14 @@
 {
-  "name": "too_many_choco",
+  "name": "react-app",
```
Comment (shiipou) [inline react-app/package.json:2]: Please don't change the React app name : 
```suggestion
  "name": "too_many_choco",
```

**Review by shiipou — CHANGES_REQUESTED** [review]
PR Title must be in type `feat` for this PR.

feat: mean a new feature
fix: mean a bug fix
build: mean a change to the build.gradle or similar things
style: mean changing code style (change some variables names or remove a space in code)
...

You can see at the conventionnal commit website for more info.

**Review by shiipou — CHANGES_REQUESTED** [review]
Can you add some documentations about your changes ? In the directory `docs/`just create a readme.md explaining how your calendar work for future people that can work on it.

**Review by Omar1H1 — CHANGES_REQUESTED** [review]

**Review by ThibaultMesmin — APPROVED** [review]

**Review by Omar1H1 — APPROVED** [review]

**Review by corantoine — APPROVED** [review]

**Review by OlivierV79 — APPROVED** [review]

**Review by CharlotteCharrier — APPROVED** [review]

**Review by JCMaif — APPROVED** [review]

**Review by jebourdeau — APPROVED** [review]

**Review by shiipou — APPROVED** [review]

**Pull request merged**